src/test/init/features/project.spec.ts

```diff
@@ -36,9 +36,12 @@ const TEST_PROJECT_INFO: ProjectInfo = {
 
 describe("project", () => {
   const sandbox: sinon.SinonSandbox = sinon.createSandbox();
-  let listProjectsStub: sinon.SinonStub;
-  let getProjectStub: sinon.SinonStub;
-  let promptStub: sinon.SinonStub;
+  let listProjectsStub: sinon.SinonStub<
+    [string?, firebaseApi.FirebaseProject[]?],
+    Promise<firebaseApi.FirebaseProject[]>
+  >;
+  let getProjectStub: sinon.SinonStub<[string], Promise<firebaseApi.FirebaseProject>>;
+  let promptStub: sinon.SinonStub<[prompt.Question], Promise<string>>;
```

Comment:
Noticed all these brackets around the first arg in the type - assuming this is just syntax for separating arg type from return type?

Reply:
correct. (see the other comment).